src/corefx/System.Globalization.Native/localeNumberData.cpp

```diff
@@ -391,3 +462,44 @@ extern "C" int32_t GetLocaleInfoInt(const UChar* localeName, LocaleNumberData lo
 
 	return UErrorCodeToBool(status);
 }
+
+/*
+PAL Function:
+GetLocaleInfoInt
+
+Obtains integer locale information
+Returns 1 for success, 0 otherwise
+*/
+extern "C" int32_t GetLocaleInfoGroupingSizes(const UChar* localeName, LocaleNumberData localeGroupingData, int32_t* primaryGroupSize, int32_t* secondaryGroupSize)
+{
+	Locale locale = GetLocale(localeName);
+	if (locale.isBogus())
+	{
+		return UErrorCodeToBool(U_ILLEGAL_ARGUMENT_ERROR);
```

Comment:
That was a bit of scaffolding for some better error handling. Like for possible SetLastError support or to do some global asserts on errors that shouldn't happen. I'm going to do the latter assert checking in next changes